ui/src/utils/numberFormats.ts

```diff
@@ -0,0 +1,11 @@
+export const bytesToMB = (bytes: number) => bytes / (1024 * 1024)
+
+export const getTotalLabel = (
```

Comment:
I thought this was used in multiple places so I moved to utils! But then realized it was only in on place. But maybe its still helpful to start a utils module for number formatting?

Reply:
Yes that's great! I start adding some unit tests for these functions as well